app/backend/src/User/model/UserModel.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  model as mongooseCreateModel,
+  Schema,
+  Document,
+} from 'mongoose';
+import User from '../schemas/User';
+import MongoModel from '../../abstractClasses/model';
+// import { any } from 'zod';
+
+interface UserDocument extends User, Document {}
```

Comment:
what you think of moving the process of data access management  to the model class?

controller will be in charge of sending the proper status codes and body
service can be in charge of the business rules 
model can instantiate the model and access the DB